CHANGES

```diff
@@ -1,6 +1,7 @@
 243
 
 - Use pipe as separator for openmetrics object names config
+- Ensure each ObjectMapperProvider has its own JsonFactory object
```

Comment:
We can also say "fix deserializing JSON with large keys"

Reply:
hya